### DISCUSSION TITLE
[A-L] (2023/24) Foglio 7 - Esercizio 8

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/calcolo-delle-probabilita#190] [Esercizi - Bertini]
![Screenshot 2023-11-13 194740](https://github.com/sapienzastudentsnetwork/calcprob2324/assets/107944476/8e0f7eb6-b372-4898-bd81-4262d05dc6a8)

**Comment by cornflexxx** [comment]
Come ci viene suggerito possiamo ragionare in termini di variabili aleatorie geometriche , quindi chiamiamo $X$ la $v.a$ che conta il numero di giorni necessari al completamento dell'album. Sappiamo che per completare l'album dobbiamo trovare tutte le figurine ,  se indichiamo  $X_i$ con $i=0,...,n-1$ la $v.a.$ che stabilisce il numero di giorni per trovare  la $i$-esima carta, allora possiamo riscrivere $X$ ne seguente modo : 
$$X=\sum_{i=0}^{n-1} X_i$$
Sappiamo che la distribuzione geometrica ha la seguente notazione : $\mathbb{P}(X_i=k)=p_i(1-p_i)^{k-1}$, in questo caso stiamo dicendo che  aggiungeremo una carta nuova alla nostra collezione con una probabilità di successo $p_i$, dopo aver trovato per $k-1$ giorni dei doppioni ( con prob. $1-p_i$). 
In generale la probabilità di trovare l' $i$-esima carta  è : 
$$p_i=\frac{n-i}{n}$$
ll risultato si giustifica in modo molto semplice dato che inizialmente abbiamo 0 carte della collezione e sicuramente la prima che compriamo avrà $p_0= 1 = \frac{n}{n}$, procedendo con la seconda $p_2=\frac{n-1}{n}$ e così via fino a $p_{n-1}=\frac{1}{n}$ (probabilità di trovare l'ultima carta della collezione).

$1)$ Trovandoci in una distribuzione geometrica (di v.a.) il valore atteso è semplicemente:
$$\mathbb{E}[X_i] = \frac{1}{p_i}$$
in questo caso:

$$\mathbb{E}[X_i] = \frac{n}{n-i}\qquad(I)$$
Per linearità del valore atteso abbiamo che : 
$$\mathbb{E}[X] =\mathbb{E}[\sum_{i=0}^{n-1} X_i]=\sum_{i=0}^{n-1} \mathbb{E}[ X_i]$$

Sostituisco $(I)$:
$$\sum_{i=0}^{n-1} \frac{n}{n-i}=n\sum_{i=0}^{n-1} \frac{1}{n-i}=n[\frac{1}{n}+\frac{1}{n-1}+...+\frac{1}{2}+1]\qquad (II)$$

$2)$ Possiamo riscrivere $(II)$ come: 
$$\sum_{i=1}^{n} \frac{1}{i}\qquad(III)$$
Poniamo $a=1,b=n,x=i$ , sappiamo che $f(x)=\frac{1}{x}$ è limitata e continua nell'intervallo $[a,b]$, questo  ne implica l'integrabilità.  Sappiamo che la $(III)$ approssima il valore dell'area presente al di sotto del grafico della funzione $f(x)$ nell'intervallo $[a,b]$. Diciamo quindi che la $(III)$ rappresenta la somma di Riemman in cui si prendono degli intervalli  di grandezza $1$ (prendiamo $x_i-x_{i+1}=1$ $\forall i\in[a,b]$):
$$\sum_{i=1}^{n} \frac{1}{x}=\sum(f,D)\qquad\text{dove D sta per la dimensione massima  di intervallo che prendiamo}$$
Le somme integrali superiori e inferiori servono a fare una stima per eccesso o per difetto dell'area sottesa dal grafico della funzione, quindi : 
$$s(f,D)\le\int_{a}^{b}f(x)dx\le S(f,D)\qquad\text{s,S stanno per somme integrali inf/sup}$$

Diciamo che più la dimensione $D$, del singolo intervallo, è piccola più le due somme integrali inferiori e superiori si avvicinano tra loro  , e quindi si avvicinano di conseguenza all'integrale.
Per questo motivo possiamo approssimare la $(III)$ con l'integrale definito in $[1,n]$ di $f(x)$.

$$\sum_{x=1}^{n}\frac{1}{x}\approx (\int_{1}^{n}\frac{1}{x}dx)+\varepsilon=[F(n)-F(1)]+\varepsilon=[log(n)-log(1)]+\varepsilon=log(n)+\varepsilon$$

$\varepsilon$  va considerato come l'errore di approssimazione , ma non so bene come procedere per calcolarlo.
